regen-1/genesis.tmpl.json

```diff
@@ -167,14 +166,14 @@
         "inflation_max": "0.200000000000000000",
         "inflation_min": "0.070000000000000000",
         "goal_bonded": "0.670000000000000000",
-        "blocks_per_year": "6311520"
+        "blocks_per_year": "4360000"
```

Comment:
This should be caluclated based off of average block-time. "6311520" is the blocks_per_year for a 5s average block time (which is also what cosmos hub started with when they launched). On our prelaunch testnet we have an avg block time of 5.4s, whereas on cosmos hub the current avg block time is 7.2s.

I don't think we should necessarily set the same value here as the hub unless we actually expect our blocktime to be a pretty close match to that of the hub.

Here's a link to the hub gov proposal where they changed it from 6311520 initially: https://forum.cosmos.network/t/governance-adjustment-of-blocks-per-year-to-come-aligned-with-actual-block-time/1682

On deeper research it seems like avg block time is something that we will only really know once our chain is running with our live validator set. I'm fine with airing on the side of caution (preferring a lower rather than higher blocks per year, as i'd rather inflation rewards be higher as opposed to lower than expected -- @glandua has mentioned this desire separately when we initially were considering a quarterly inflation adjustmenet period).

So maybe we do stick with the conservative number (4360000) to mirror cosmos hub, but we should track this and probably have a gov vote for a param change to set it to our actual `blocks_per_year` once that number stabilizes?

Reply:
Makes sense